src/popup/modules/UserInterface.js

```diff
@@ -184,13 +174,7 @@ function scrollToTop(event) {
         return;
     }
 
-    // Attention: make sure this does not collide with the retry-property set
-    // in selectAllText()!
     event.setScrolled = true;
```

Comment:
If we don't even re-try scrolling to the top, I guess we can remove all that `setScrolled` logic too, can't we?

Reply:
@rugk I agree! But I think there is still a setTimeout for that one [here](https://github.com/cornu-ammonis/offline-qr-code/blob/da9bdeb100dbb3cf028c19ba71dbb3b862a1c2e8/src/popup/modules/UserInterface.js#L153), should we take that out too?

Reply:
That's a good question and I am not sure. Is this still needed? Possibly not…?

Or possibly it is, considering "scroll to top" does not work anyway always in the current situation in my testing at least.

Reply:
@rugk yeah, seems like that may call for a larger refactor

Reply:
Okay, fine, accepted. Also opened a different issue for that now: https://github.com/rugk/offline-qr-code/issues/203